test/extended/quota/resourcequota.go

```diff
@@ -164,6 +164,58 @@ var _ = g.Describe("[sig-api-machinery][Feature:ResourceQuota]", func() {
 			})
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
+
+		g.It("check the quota after import-image with --all option", func() {
+			testProject := oc.SetupProject()
+			testResourceQuotaName := "my-imagestream-quota-" + testProject
+			clusterAdminKubeClient := oc.AdminKubeClient()
+
+			rq := &corev1.ResourceQuota{
+				ObjectMeta: metav1.ObjectMeta{Name: testResourceQuotaName, Namespace: testProject},
+				Spec: corev1.ResourceQuotaSpec{
+					Hard: corev1.ResourceList{
+						"openshift.io/imagestreams": resource.MustParse("10"),
+					},
+				},
+			}
+
+			g.By("create the imagestreams and checking the usage")
+			_, err := clusterAdminKubeClient.CoreV1().ResourceQuotas(testProject).Create(context.Background(), rq, metav1.CreateOptions{})
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			err = waitForResourceQuotaStatus(clusterAdminKubeClient, testResourceQuotaName, testProject, func(actualResourceQuota *corev1.ResourceQuota) error {
+				expectedUsedStatus := corev1.ResourceList{
+					"openshift.io/imagestreams": resource.MustParse("0"),
+				}
+				if !equality.Semantic.DeepEqual(actualResourceQuota.Status.Used, expectedUsedStatus) {
+					return fmt.Errorf("unexpected current total usage: actual: %#v, expected: %#v", actualResourceQuota.Status.Used, expectedUsedStatus)
+				}
+				return nil
+			})
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("trying to tag a container image")
+			err = oc.AsAdmin().WithoutNamespace().Run("import-image").Args("centos", "--from=quay.io/openshifttest/alpine", "--confirm=true", "--all=true", "-n", testProject).Execute()
```

Comment:
Nit: sometime in the future if that repo gets a new tag, this might start failing, so ideally it would be good to either expect min. 2 images, rather than exactly 2. Or even better explicitly import specific 2 tags, and then check  the number. But that's a minor problem that can be fixed in a followup.